### ISSUE TITLE
Make blocking ads not the default option (creators are losing money from people who might not want them to)

### ISSUE THREAD

**Issue by markipol** [YePpHa/YouTubeCenter#335]
So anyway, when you first install youtube center (on chrome) the "Block Advertisement" option is on. _possible preachy-ness incoming_ Youtube content creators don't recieve money based on views on a video, it's based on how many have _actually_ seen the ad on the video. I am not saying don't get rid of the blocking of the ads, just make it so it must be enabled. The stated intention of this extension is to add a download button and all other sorts of options, and people may not realise ad blocking is a feature. Some people may not want to change anything in the settings menu, and you have to, to stop ad blocking, because they just want the bar above the video desc. for downloads, quality etc. therefore they will be blocking ads and they won't notice. The reason I am writing this is I had gone the previous week or so without ads accidentally (not turning it off), depriving my favourite youtubers of revenue.

tl;dr Please make blocking ads a feature that's not turned on by default so that people won't accidentally deprive creators of revenue.
